vmware-event-router/internal/integration/openfaas_test.go

```diff
@@ -104,32 +99,19 @@ var _ = Describe("OpenFaaS Processor", func() {
 			It("should receive a failed invocation response", func() {
 				Expect(failCount).To(Equal(1))
 			})
-
-			// OpenFaaS processor does not return error directly, only in case
-			// of JSON marshaling issues which we don't expect for this test case
-			It("should not error", func() {
-				Expect(err).NotTo(HaveOccurred())
-			})
 		})
 
 		Context("when no function in OpenFaaS is subscribed to that event type (LicenseEvent)", func() {
-			// create LicenseEvent and marshal to CloudEvent
 			BeforeEach(func() {
 				baseEvent = newLicenseEvent()
 				ce, err = events.NewCloudEvent(baseEvent, fakeVCenterName)
 				Expect(err).ShouldNot(HaveOccurred())
 			})
 
-			// process and give response time to get back from OpenFaaS gateway
-			// (note: we don't expect response, just making sure nothing gets
-			// through)
 			BeforeEach(func() {
 				err = ofProcessor.Process(ctx, *ce)
-				time.Sleep(waitDelay)
```

Comment:
Why did we sleep before? Was it precaution, a second to process just in case?

Reply:
Because we did not wait for `InvokerResponse` (that was fixed with retries)

Reply:
Yeah ok, I was asking since I saw it here, thanks for the clarification 👍